pulp_ansible/app/tasks/collections.py

```diff
@@ -576,11 +576,15 @@ async def _add_collection_version(self, api_version, collection_version_url, met
         )
         cv_unique = attrgetter("namespace", "name", "version")(collection_version)
         fullname, version = f"{cv_unique[0]}.{cv_unique[1]}", cv_unique[2]
-        if fullname in self.exclude_info and Version(version) in self.exclude_info[fullname]:
-            return
         if cv_unique in self.already_synced:
             return
+
+        # Mark the collection version as being processed
         self.already_synced.add(cv_unique)
```

Comment:
What is this? Apart from eating a lot of memory?

Reply:
Do we want to skip this when doing a full mirror sync anyway?
What if we do a bloom filter thing?

(Just a few thoughts...)

Reply:
Sorry, this was asking on a tangent. Would you mind making it a separate change?